src/twisted/web/newsfragments/12104.bugfix

```diff
@@ -0,0 +1,2 @@
+The dictionary twisted.web.http.RESPONSES correctly returns `I'm a teapot`
+as default message for the status code 418.
```

Comment:
Just a minor suggestion to comply with the project policy.

The imporant API change here is the public `twisted.web.http.IM_A_TEAPOT` API

I am leaving this as a bug, since we have not fully implemented the RFC :p


```suggestion
twisted.web.http.IM_A_TEAPOT was added and returns `I'm a teapot`
as default message for the status code 418,
as defined in RFC 2324 section 2.3.2.
```